GOVERNANCE.md

```diff
@@ -12,6 +12,8 @@ ResBaz is committed to the following values:
 
 ## Governance
 
+***TODO: this is wrong!!!***
```

Comment:
this is wrong!

Reply:
@lossanna raised a really good point IRL: "are we elected?"

Reply:
No. As far as I can recall there have never been elections.  @bjoyce3?

Reply:
Which is not to say there _shouldn't_ be elections.